index.js

```diff
@@ -30,15 +30,14 @@
 
 'use strict';
 
-var Handlebars = require('./lib/vendors/handlebars'),
-	TemplateProvider = require('./lib/TemplateProvider');
+const Handlebars = require('./lib/vendors/handlebars');
+const TemplateProvider = require('./lib/TemplateProvider');
 
 module.exports = {
-	register: function (locator, config) {
-		config = config || {};
+	register(locator, config) {
```

Comment:
`config` should be resolved from the locator. We shouldn't have the `config` parameter anymore.